EN/Moodles_EN.txt

```diff
@@ -54,14 +54,14 @@ Moodles_EN = {
 	Moodles_angry_desc_lvl4 = "90%^ Angry<br>no effects",
 	-- Stress
 	Moodles_stress_desc_lvl1 = "10%^ Stressed<br>foraging vision reduced",
-	Moodles_stress_desc_lvl2 = "25%^ Stressed<br>less foraging vision<br>-10% melee damage<br>unhappiness increase",
-	Moodles_stress_desc_lvl3 = "50%^ Stressed<br>even less foraging vision<br>-20% melee damage<br>unhappiness increases fast",
-	Moodles_stress_desc_lvl4 = "75%^ Stressed<br>worse foraging vision<br>-30% melee damage<br>unhappiness quickly increases",
+	Moodles_stress_desc_lvl2 = "25%^ Stressed<br>less foraging vision<br>-0.1 damage<br>unhappiness increase",
+	Moodles_stress_desc_lvl3 = "50%^ Stressed<br>even less foraging vision<br>-0.2 damage<br>unhappiness increases fast",
+	Moodles_stress_desc_lvl4 = "75%^ Stressed<br>worse foraging vision<br>-0.3 damage<br>unhappiness quickly increases",
```

Comment:
Revert these since it makes it unclear as people are used to percentages not scaling values, which is also what you pointed out earlier in your first changes.
A float value of 0.10 in a multiplication equals 10%, but the latter is easier to grasp.

Reply:
it does not go through a multiplication though, hence the modification, it's a flat change in weapon damage

Reply:
well it does get multiplied by other stuff, but it's the damage of the weapon who gets substracted then multiplied by other damage reducing factors ( that's my understanding of the piece of code )

Cf SwipeStatePlayer lines 1393 (declaration of var 34, which gets pretty complicated)

Modificators are first applied lines 1466-1481 to apply traits modification (underweight ...)
Moodles are then applied lines 1507-1519 and are substraction of the raw damage value
At this step the dmg is set to be at least 0.1 (so that hitting a zombie can't do nothing if it suceeds)
Additionnal moodles for melee are then applied lines 1536-1577 as multiplicator for Tired/Endurance to take effect
Meaning at that stage the minimal damage dealt is 0.1*0.05*0.05 (if you're max tired, and with no stamina)
There are some more math after that but I was not patient enough to try to understand what the hell all those variables are

Reply:
this means, the damage is in fact reduced by 0.1 and not 10%, because 10% could be much more than 0.1 (or much less)

Reply:
I went to check, and that seems to be correct. Up to that point it kinda just slowly checks and scales, then subtracts and adds small values to the variable, makes sure it never goes under 0.1, then scales it, does the "hitbox" calculation, and eventually... randomizes it based on another value related to defense??? Honestly, no shame that you stopped calculating, haha.

And after reading all of those mistakes you pointed out, once again, this is another reason why I shouldn't do things at like 3 AM. LOL

But again, it is appreciated that you went through the effort of double and triple-checking things that I missed, more heads always help to think and get feedback.